### ISSUE TITLE
[HW2] comment 관련 질문드립니다.

### ISSUE THREAD

**Issue by jihoonrf** [swsnu/swppfall2019#66]
스펙 문서에는 comment와 관련된 component들(edit comment btn, delete comment btn)이 id값 하나만이 존재하는 유니크한 component인 것처럼 서술돼 있는데 맞게 이해한 건지 궁금합니다.
만약 그렇다고 한다면 한명의 유저는 하나의 article에 대해 최대 하나의 comment만을 작성할 수 있어야 할 것 같은데 이런 컨셉이 맞나요?

추가로, 최대 하나의 comment가 맞다면 각 component 자체도 조건부로 랜더링되어야 하는 것이 직관적일 것 같은데 (로그인된 유저가 작성한 comment가 있다면 confirm 버튼은 표시하지 않고, 반대로 comment가 없다면 edit, delete 버튼은 표시하지 않는 식으로) 이 부분도 어떻게 처리해야 할지 여쭙고 싶습니다.

**Comment by ktaebum** [comment]
네 개념적으로는 `id`는 unique해야하는 게 맞습니다. 하지만 우선 chrome에선 multiple id를 허용하기도 하고, 일단 id로 구분한 건 저희가 채점하기 편하게 하기 위해서 그렇게 하였습니다.

따라서 그 부분은 신경 안 쓰셔도 되고, 한 명의 유저가 여러 커멘트도 허용하는 것이라고 생각하시면 됩니다
